### ISSUE TITLE
Install by Homebrew does not work

### ISSUE THREAD

**Issue by GenoZhou** [AliSoftware/SwiftGen#177]
Run `brew install swiftgen` returns following error under Xcode8:
```
rake aborted!

[!!!] You need to use Xcode 7.x to compile SwiftGen. Use xcode-select to change the Xcode used to build from command line.

/private/tmp/swiftgen-20160926-78163-1b2b956/SwiftGen-3.0.0/Rakefile:46:in `block in <top (required)>'
Tasks: TOP => install => install:light => build => check_xcode_version
(See full trace by running task with --trace)
```

**Comment by highthem** [comment]
same issue here

**Comment by AliSoftware** [comment]
You could just follow the instructions/suggestion given by the error message.

Anyway there's already a PR pending to migrate to Xcode 8, so I'm gonna close this duplicate issue.

**Comment by AliSoftware** [comment]
Fixed by #179.

SwiftGen 3.0.1 should be installable by Homebrew properly as soon as Homebrew/homebrew-core#6663 is merged by the Homebrew team!
